distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -12,14 +12,17 @@ polyglot java import org.enso.table.error.EmptySheetException
 type Missing_Input_Columns
     ## PRIVATE
        One or more columns not found in the input table.
-    Error (criteria : [Text])
+    Error (criteria : [Text]) (where:Text|Nothing = Nothing)
 
     ## PRIVATE
 
        Convert a missing input error to a human-readable form.
     to_display_text : Text
     to_display_text self =
-        "The criteria "+self.criteria.to_text+" did not match any columns."
+        where = case self.where of
+            Nothing -> "."
+            location : Text -> " in "+location+"."
+        "The criteria "+self.criteria.to_text+" did not match any columns"+where
```

Comment:
Need a space after "columns".

Reply:
No, if `where` is `.` the space should not be there, and in the other branch the space is added to `where`.